phalcon/mvc/model/criteriainterface.zep

```diff
@@ -192,6 +193,6 @@ interface CriteriaInterface
 	/**
 	 * Executes a find using the parameters built with the criteria
 	 */
-	public function execute() -> <ResultsetInterface>;
+	public function execute(<TransactionInterface> transaction = null) -> <ResultsetInterface>;
```

Comment:
hmm I understand, it just would have been nice not to be wasting so much of our time with basically style changes when this fact/functionality was already clear an visible with my first commit of the PR.

anyways I have a lot of projects with deadlines, so today was probably my last chance to finish this within the next 3-5 weeks.

please let me know what you guys decide.